src/main/java/com/kodilla/ecommercee/domain/Group.java

```diff
@@ -0,0 +1,14 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+public class Group {
```

Comment:
Czy ta klasa jest potrzebna na tym etapie?

Reply:
W sumie na tym etapie to nie jest potrzebna klasa Group, będzie potrzebna podczas mapowania obiektu klasy GroupDto na obiekt klasy Group.

Reply:
Zgadza się, Group to encja, usuń ją proszę i sprawdź CircleCI